doc/documentation_contributors.md

```diff
@@ -0,0 +1,116 @@
+Guideline for contributing documentation
+========================================
+The DLite documentation is written in Markdown.
+This include both the README files and documentation found in the `doc/` subdirectory.
+
+Common to both is that the text should be as easy and natural as possible to read from the terminal.
+Hence, the following recommendations should be followed:
+
+* Write one sentence per line, in order to get an easier to read output from `git diff`.
+
+* For README files, use the underline style format for main and sub-headers.
+  For example, start a new page with
+
+      Overview
+      ========
+
+  instead of
+
+      # Overview
```

Comment:
```suggestion
  The setext header styling:

  ```markdown
  Header level 1
  ==============

  Some text.

  Header level 2
  --------------

  Some text.
  ```
```

Reply:
Reference to heading style: [MD003](https://github.com/DavidAnson/markdownlint/blob/main/doc/md003.md).